app/config/ServiceConfiguration.scala

```diff
@@ -102,4 +113,9 @@ class PlayBasedServiceConfiguration @Inject()(configuration: Configuration) exte
         throw new Exception(s"Configuration key not correctly configured: $key, errors: ${errors.mkString(", ")}")
     }
   }
+
+  override def defaultAllowedMimeTypes: List[String] =
+    configuration.getString("fileTypesFilter.defaultAllowedMimeTypes")
+      .map(_.split(",").toList)
+      .getOrElse(Nil)
 }
```

Comment:
Is this safe, or is it not a significant configuration error if there is not at least one default mime type?
Just worried about the ease of rolling something out that is misconfigured and simply blocks everything.